### PR TITLE
fix next/prev month button onkeydown handlers

### PR DESCRIPTION
**Pull request by Abdubek**

### CODE REVIEW

**Comment by Abdubek** [comment]
is there any news about the acceptance of this pull request?

**Comment by yuliia007naumenko** [comment]
Are there any updates on pull request? A also need ongoing changes

**Comment by Abdubek** [comment]
@martijnrusschen Any thoughts on this pull request?

File: src/calendar.jsx [hunk 1]
```diff
@@ -180,7 +180,7 @@ export default class Calendar extends React.Component {
     onMonthMouseLeave: PropTypes.func,
     showPopperArrow: PropTypes.bool,
     handleOnKeyDown: PropTypes.func,
-    handleTimeKeyDown: PropTypes.func,
```
[the remark this answers is not in the document]
Comment (Abdubek) [reply src/calendar.jsx:183]: this change does not break the end user api, I just changed the name to a more correct one within this task, improving the understanding of the code

i can redo, with minimal code, without changing the name of props, let me know if needed

**Pull request merged**